### ISSUE TITLE
Get rid of deployment specific container postfixes

### ISSUE THREAD

**Issue by hbruch** [verschwoerhaus/digitransit-kubernetes#1]
I wonder if it wouldn't be easier to get rid of all the -vsh etc postfixes for containers, assuming that one cluster would only serve a single region deployment. As OTP even plans to drop support for multiple routers per instance, there would be no need to deviate from the default router neither.

That way, only a few images would be region specific. If there repo-paths are extracted as variables, the whole setup would be more portable.

WDYT?

**Comment by robbi5** [comment]
yep. this is exactly the conclusion we came to yesterday too. i'll try to introduce kubernetes namespaces and going back to postfix-free services/containers in the next few days :)

**Comment by robbi5** [comment]
It finally happened (in https://github.com/stadtulm/digitransit-k8s that was resolved months ago).
Maybe the next step would be to extract the images and image tags with kustomize, [like codeformuenster does]?(https://github.com/codeformuenster/kubernetes-deployment/blob/0fd4fbd3b8c899230b1727275dfef5ae85127f00/sources/digitransit/kustomization.yaml#L19)
